flycheck.el

```diff
@@ -312,6 +312,22 @@ respectively."
   :safe #'flycheck-symbol-list-p)
 (make-variable-buffer-local 'flycheck-disabled-checkers)
 
+(defvar-local flycheck-automatically-disabled-checkers nil
```

Comment:
`flycheck--` instead of `flycheck-`, maybe?

Reply:
I forget we are using this convention now.  :+1: